Build/Tactical/TeamTurns.cc

```diff
@@ -146,6 +148,9 @@ void StartPlayerTeamTurn( BOOLEAN fDoBattleSnd, BOOLEAN fEnteringCombatMode )
   /// that a baddie can die and still maintain it's attacker ID
   gTacticalStatus.fKilledEnemyOnAttack = FALSE;
 
+  // Save if we are in Dead is Dead mode
+  if (gGameOptions.ubGameSaveMode == DIF_DEAD_IS_DEAD)   DoDeadIsDeadSave();
```

Comment:
Or create a new function that wraps it and inline it there: `DoDeadIsDeadSaveIfNecessary`